systemd/os-autoinst-scripts-update-git.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Update git repository of os-autoinst-scripts
+After=network-online.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/git -C /opt/os-autoinst-scripts pull --quiet --rebase origin
+User=geekotest
+
+[Install]
+WantedBy=default.target
```

Comment:
You might want to drop the install section as this service is supposed to be started only with the timer and not actually enabled?